joss_paper/paper.md

```diff
@@ -0,0 +1,121 @@
+obtaining scientific insights from molecular dynamics trajectory data. 
+Instead, the ability to process, analyze and share large volumes of data provide 
+new constraints on research in this field.
+
+Other groups in the field recognize this same need for adherence to 
+FAIR principles [@FAIR:2019] including the MDDB (Molecular Dynamics Data Bank), an EU-scale 
+repository for biosimulation data [@MDDB:2024] and MDverse, a prototype search engine 
+for publicly-available Gromacs simulation data [@MDverse:2024].
+While these efforts currently offer prototype solutions for indexing and 
+searching MD trajectory data, the problem of efficiently distributing the data remains. 
+
+Though exposing download links on the open internet offers a simple solution to this problem,
+on-disk representations of molecular dynamics trajectories often range in size 
+with large datasets up to TBs in scale [@ParallelAnalysis:2010] [@FoldingAtHome:2020],
+so a solution which could prevent this 
+duplication of storage and unnecessary download step would provide greater utility 
+for the computational molecular sciences ecosystem.
+
+Enter `Zarrtraj`, the first fully-functioning tool to our knowledge that allows 
+streaming trajectories into analysis software using an established trajectory format.
+`Zarrtraj` is implemented as an `MDAnalysis` [@MDAnalysis:2016] `MDAKit` [@MDAKits:2023] that
+enables streaming MD trajectories in the popular `HDF5`-based H5MD format [@H5MD:2014]
+from AWS S3, Google Cloud Buckets, and Azure Blob Storage & Data Lakes without ever downloading them.
+This is possible thanks to the `Zarr` [@Zarr:2024] package which allows 
+streaming array-like data from a variety of storage mediums and [Kerchunk](https://github.com/fsspec/kerchunk), 
+which extends the capability of `Zarr` by allowing it to read `HDF5` files.
+Because it implements the standard `MDAnalysis` trajectory reader API,
+`Zarrtraj` can leverage `Zarr`'s ability to read a file in parallel to perform analysis 
+algorithms in parallel using the "split-apply-combine" paradigm. In addition to the `H5MD` format, 
+`Zarrtraj` can stream and write trajectories in the experimental `ZarrMD` 
+format, which ports the `H5MD` layout to the `Zarr` filetype.
+
+One imported, `Zarrtraj` allows passing trajectory URLs just like ordinary files:
+```python
+import zarrtraj
+import MDAnalysis as mda
+
+u = mda.Universe("sample_topology.top", "s3://sample-bucket-name/trajectory.h5md")
+```
+Initial benchmarks show that `Zarrtraj` can iterate
+through an AWS S3 cloud trajectory (load into memory one frame at a time)
+at roughly 1/2 or 1/3 the speed it can iterate through the same trajectory from disk and roughly 
+1/5 to 1/10 the speed it can iterate through the same trajectory on disk in XTC format \autoref{fig:benchmark}.
+However, it should be noted that this speed is influenced by network latency and that
+writing parallelized algorithms can offset this loss of speed.
```

Comment:
I wouldn't put out bold claims with evidence. If you can show an example of "parallelized algorithms" then explain how and show data to back up your claim. Otherwise leave it out.

Basic rule of academic writing: When you write a statement you back it up
* by showing your own data
* by citing someone else's work

Otherwise you leave it out or mention it at the very end as possible avenues for future work.